imageroot/systemd/user/mattermost-app.service

```
@@ -0,0 +1,33 @@
+[Unit]
+Description=Podman  mattermost-app.service
+BindsTo=mattermost.service
+After=postgres-app.service
+
+[Service]
+Environment=PODMAN_SYSTEMD_UNIT=%n
+EnvironmentFile=%S/state/environment
+WorkingDirectory=%S/state
+Restart=always
+TimeoutStopSec=70
+ExecStartPre=/bin/rm -f %t/mattermost-app.pid %t/mattermost-app.ctr-id
+ExecStart=/usr/bin/podman run --conmon-pidfile %t/mattermost-app.pid \
+    --cidfile %t/mattermost-app.ctr-id --cgroups=no-conmon \
+    --pod-id-file %t/mattermost.pod-id --replace -d --name  mattermost-app \
+    --volume mattermost-config:/mattermost/config:Z \
+    --volume mattermost-data:/mattermost/data:Z \
+    --volume mattermost-plugins:/mattermost/plugins:Z \
+    --volume mattermost-plugins-client:/mattermost/client/plugins:Z \
+    --volume mattermost-bleve-indexes:/mattermost/bleve-indexes:Z \
+    --env MM_SQLSETTINGS_DRIVERNAME=postgres \
+    --env MM_SQLSETTINGS_DATASOURCE=postgres://mmuser:mmuser_password@127.0.0.1:5432/mattermost?sslmode=disable&connect_timeout=10 \
+    --env TZ=UTC \
+    --env MM_SERVICESETTINGS_SITEURL=${MM_SERVICESETTINGS_SITEURL} \
+    ${MATTERMOST_TEAM_EDITION_IMAGE}
+ExecStop=/usr/bin/podman stop --ignore --cidfile %t/mattermost-app.ctr-id -t 10
+ExecReload=/usr/bin/podman kill -s HUP mattermost-app
```

`SyslogIdentifier=%u` for rootless modules helps to group the module log lines together in the cluster-admin/log page.

```suggestion
ExecReload=/usr/bin/podman kill -s HUP mattermost-app
SyslogIdentifier=%u
```